src/cf-forms.less

```diff
@@ -448,6 +416,86 @@ textarea {
 
 }
 
+/* topdoc
+  name: Notifications
+  family: cf-notifications
+  notes:
+    - "Markup and styling for notifications"
+  patterns:
+    - name: Default Usage
+      markup: |
+        <div class="cf-notification cf-notification__error">
+          <span class="cf-notification_icon cf-notification_icon__error cf-icon cf-icon-error-round"></span>
+          <p class="cf-notification_text">This is the default alert.</p>
+        </div>
+  tags:
+    - cfgov-misc
+*/
+
+@notification-padding__px: 11px;
+
+.cf-notification {
+    display: none;
+    position: relative;
+
+    padding: @notification-padding__px;
+    padding-left: 40px;
+    margin-bottom: @notification-padding__px + 4px;
+
+    background: @gray-20;
+    border: 1px solid @gray;
+
+    &__visible {
+      display: block;
+    }
+
+    &__error {
+        background: @redorange-20;
+        border-color: @input-error;
+    }
+
+    &__success {
+        background: @green-tint;
+        border-color: @input-success;
+    }
+
+    &__warning {
+        background: @gold-20;
+        border-color: @input-warning;
+    }
+
+    &_icon {
+        position: absolute;
+        top: @notification-padding__px + 2px;
+        left: @notification-padding__px + 4px;
+
+        color: @gray-20;
+        font-size: unit( 18px / @base-font-size-px, em );
+
+        &__error {
+            color: #d14327;
```

Comment:
These absolute colors came from @duelj as slight tweaks to the input colors. Justin do you remember what tweaking we did? Could we automate it somehow using the input variables? Ex. `darken(@input-error, 15%)` or something similar using Less color functions.